package/kedro_viz/server.py

```diff
@@ -55,6 +55,20 @@
     __name__, static_folder=str(Path(__file__).parent.absolute() / "html" / "static")
 )
 
+ERROR_PROJECT_ROOT = (
+    "Could not find a Kedro project root. "
+    "You can run `kedro viz` by either providing `--load-file` flag with a JSON file "
+    "path for your pipeline, or if the current working directory is "
```

Comment:
Maybe " `--load-file` flag with a filepath to a JSON pipeline representation, ..."